src/overlays/actors/ovl_En_Bw/z_en_bw.h

```diff
@@ -7,11 +7,27 @@
 struct EnBw;
 typedef void (*EnBwActionFunc)(struct EnBw*, PlayState*);
 
+typedef enum {
+    /*  0 */ TORCH_SLUG_LIMB_NONE,
+    /*  1 */ TORCH_SLUG_LIMB_BODY,
+    /*  2 */ TORCH_SLUG_LIMB_LEFT_EYESTALK_ROOT,
+    /*  3 */ TORCH_SLUG_LIMB_LEFT_EYESTALK_WRAPPER,
```

Comment:
We don't have an agreed-upon way to handle these wrapper limbs, but this is yet another variant to handle them,

afaik the repo currently has
`_ROOT1`, `_ROOT2` (BossGoma, EnGoma)
`_ROOT_ROOT`, `_ROOT` (EnZf)

Personally I'd like either of those approach over `_ROOT`, `_WRAPPER`, it seems easier to tell what the parent is

Thoughts?

Reply:
This scheme comes from MM, but since I'm playing in OoT's sandbox, I can acquiesce. I'll switch to doing what EnZf does.

Reply:
I didn't know MM used this. I'll need to read why this way was chosen